shaders/src/common_types.fs

```diff
@@ -34,3 +34,8 @@ struct ShadowData {
     float bulbRadiusLs;
     float nearOverFarMinusNear;
 };
+
+struct BoneData {
+    highp mat3x4 transform;    // bone transform is mat4x3 stored in row-major (last row [0,0,0,1])
```

Comment:
Won't we run into the adreno bug with highps inside struct being ignored?

Reply:
I'm thinking not because it's in an UBO?

Reply:
Ah right yeah that's fine.